ground-ingest/src/main/java/edu/berkeley/ground/ingest/FileMetadataSource.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
This isn't a part of LNKD. Is this something that's internal to LinkedIn? If so, then we need to make sure that it should be in OSS.